app/Models/Profile.php

```diff
@@ -33,9 +34,17 @@ public function tags()
     }
 
     /**
+     * Return the full name of the profile
+     */
+    public function full_name()
+    {
+        return "$this->first_name $this->last_name";
+    }
+
+    /**
      * Add a skill
      */
-    public function add_tag($tag, $rating=1, $is_seeking = false, $is_offering)
+    public function add_tag($tag, $rating=1, $is_offering, $is_seeking = false)
```

Comment:
Can we amend this method signature? The mandatory arguments should go at the start, and optional ones go at the end - otherwise the PHP parser will get a two-argument call wrong (the second parameter, which is optional, would be supplied and the third parameter which is mandatory would be missing).

Thanks :smile_cat: